cvat/apps/organizations/serializers.py

```diff
@@ -126,12 +125,7 @@ def create(self, validated_data):
             user_email = membership_data["user"]["email"]
             user = User.objects.create_user(username=user_email, email=user_email)
             user.set_unusable_password()
-            # User.objects.create_user(...) normalizes passed email and user.email can be different from original user_email
```

Comment:
Is this comment inaccurate? It seems like the issue you've described shouldn't happen if this is true.

Reply:
Got it, thanks.